gateleen-security/src/test/java/org/swisspush/gateleen/security/authorization/RoleMapperTest.java

```diff
@@ -50,114 +50,133 @@ public void setUp() {
     public void checkNoMapping(TestContext context) {
         Set<String> roles = new HashSet<>();
         roles.add("domain-user");
-        roles = roleMapper.mapRoles(roles);
-        context.assertNotNull(roles);
-        context.assertTrue(roles.size() == 1);
-        context.assertTrue(roles.contains("domain-user"));
+        Map<String, RoleMapper.MappedRole> mappedRoles = roleMapper.mapRoles(roles);
+        context.assertNotNull(mappedRoles);
+        context.assertTrue(mappedRoles.size() == 1);
+        context.assertTrue(mappedRoles.containsKey("domain-user"));
     }
 
     @Test
     public void checkMappingWithoutKeep(TestContext context) {
         Set<String> roles = new HashSet<>();
         roles.add("domain1-user");
-        roles = roleMapper.mapRoles(roles);
-        context.assertNotNull(roles);
-        context.assertTrue(roles.size() == 1);
-        context.assertTrue(roles.contains("domain1"));
+        Map<String, RoleMapper.MappedRole> mappedRoles = roleMapper.mapRoles(roles);
+        context.assertNotNull(mappedRoles);
+        context.assertTrue(mappedRoles.size() == 2);
```

Comment:
The size is now 2 because in the rolemapper resource, the _keepResultingRole_ property is true? Maybe a short inline comment whould help to understand the target of the test.